docs/Getting Started/openSUSE/index.rst

```diff
@@ -0,0 +1,35 @@
+.. highlight:: sh
```

Comment:
In general (not specific to any line, so I just tagged this on the first), you should look at the git log for the Debian/Ubuntu instructions (whatever you based this on) and merge in recent changes I've made.

You should probably also squash the commits together.

For the other review comments, I've only looked at the one file. I assume the other two have similar issues.

Reply:
Ok. Im working on it.